tests/test_registry.py

```diff
@@ -231,23 +502,21 @@ def _process_registration_result(self, catalog, encryptor, stage):
                 )
                 return "all done!"
 
-            def mock_do_get(self, url):
-                self.do_get_called_with = url
-                return "A fake catalog"
-
```

Comment:
The code that made the HTTP request used to be in push() (https://github.com/NYPL-Simplified/circulation/pull/1313/files#diff-4251fb997120a022b5b6c87638511e60L233), so we had to mock the request. Now push() calls Registry.fetch_catalog, and we're mocking that method, so there's no longer any reason to mock the request-making process here.